pkg/expr/classic/classic.go

```diff
@@ -184,18 +176,45 @@ func (ccc *ConditionsCmd) Execute(ctx context.Context, vars mathexp.Vars) (mathe
 	return newRes, nil
 }
 
-// UnmarshalConditionsCmd creates a new ConditionsCmd.
-func UnmarshalConditionsCmd(rawQuery map[string]interface{}, refID string) (*ConditionsCmd, error) {
+// ConditionJSON is the JSON model for a single condition in ClassicConditionsCmd.
+// It is based on services/alerting/conditions/query.go's newQueryCondition().
+type ConditionJSON struct {
+	Evaluator ConditionEvalJSON     `json:"evaluator"`
+	Operator  ConditionOperatorJSON `json:"operator"`
+	Query     ConditionQueryJSON    `json:"query"`
+	Reducer   ConditionReducerJSON  `json:"reducer"`
+	// Params []interface{} `json:"params"` (Unused)
```

Comment:
Isn't this comment actually part of the `ConditionReducerJSON`?

Reply:
Yes, oops! Thanks for reviewing 😄